### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by Floessie** [speed47/dvdisaster#16]
Salut Stéphane,

I just wanted to thank you for reviving `dvdisaster`. It's been an invaluable tool for me in the past and will sure be in the BD-R era.

I just realized it was dropped from Debian Testing for being unfit for GCC 10. Are you planning to point the maintainers to your repository so it will be part of Bullseye again? Is there any contact with Carsten Gnörlich or any info on why he abandoned `dvdisaster` without making sure someone else cares about the "official" branch?

Best,
Flössie

**Comment by speed47** [comment]
Hey Floessie, Thanks for your message.

I didn't realize dvdisaster was kicked out of Debian Testing, but you're right: [https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=957156](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=957156)
This "bug" is already fixed in my version. The patch is only a few lines long. I confirm it builds even under Debian Sid (just tried, to be sure). I'll try to push the patch to their bugtracker. As the maintainer already has TONS of custom patches for dvdisaster, hopefully (s)he'll not be against adding one more.

About the official author, he's aware of this repository, and is happy/okay with it. I think we can't say he abandoned the project, my guess is that it's mainly a life priority issue. He doesn't seem to have ruled out resuming work on it at a later time, so, only time will tell.

**Comment by Floessie** [comment]
@speed47 
> This "bug" is already fixed in my version. The patch is only a few lines long. I confirm it builds even under Debian Sid (just tried, to be sure). I'll try to push the patch to their bugtracker.

That's probably a good idea. You could also try to build it on Sid with the latest Clang. That often reveals more problems than GCC.

> He doesn't seem to have ruled out resuming work on it at a later time, so, only time will tell.

Good to know, he's aware of your work. As `dvdisaster.org` is dead this would need a reboot anyway, so he could start again based on your work, I guess.

Thanks again for your efforts,
Flössie

**Comment by speed47** [comment]
For the record, dvdisaster is back in Bullseye https://packages.debian.org/bullseye/dvdisaster

**Comment by gr01d** [comment]
I'm wondering whether you will still support this project in the near future. Just to make it work on newer OSes.

**Comment by speed47** [comment]
Yes, I'm planning to continue supporting this version for the years to come, this is the very reason of the existence of this repo. I do have media protected by dvdisaster, so it's in my direct interest to ensure it still works in 10+ years.

Note that this version does have GUI too, you just have to compile it. It's easy to provide a precompiled CLI version that works everywhere (Debian, Centos, Arch, etc) but the GUI version must be compiled on the OS it'll be running on, or it won't work, due to the multiple dependencies. Compiling yourself might be easier than you think :)

In any case, I've learnt a bit about github actions in the last few weeks, so prepackaging a GUI version for several Debian, Ubuntu etc. versions might be doable. Will look into that in the next few days

**Comment by gr01d** [comment]
One more thing, according to the manual:
```
Q 1.3: Is dvdisaster compatible with future releases?
Yes, dvdisaster files are intended for an archival time of many years. When upgrading to a
newer version of dvdisaster you can continue using images and error correction data created
from previous versions. There is no need to recreate them again.

```
does this still hold even on your version? If I create an ECC file for newer disc using the debian release of dvdisaster (0.76.5) and open them in your (0.79.6pt4) on windows (vice versa), are they still compatible?

**Comment by speed47** [comment]
AppImage might be a good idea indeed.

Yes, the compatibility does still hold with my version. To be clear, I'll never break compat with previous versions, this would be against the goal of this repo.

**Comment by speed47** [comment]
I just released an AppImage version. Thanks for the idea!

**Comment by cy7yz2rj** [comment]
Just found out about this, great news.

Upon noticing the official dvdisaster site was down I feared an eventual removal from Debian, which would be unfortunate given the purpose of the program.

Thank you for reviving it and keeping backwards compatibility and also thank you Carsten Gnörlich if you're reading.

**Comment by Yorizuka** [comment]
I don't have a lot to say except for thank you for maintaining this, this software is still critical for some of us!
